meta/main.yml

```diff
@@ -1,6 +1,13 @@
 ---
 
 dependencies:
+  - role: debops.apt_preferences
+    tags: [ 'depend::apt_preferences', 'type::dependency', 'depend-of::gitlab', 'type::dependency' ]
+    apt_preferences_dependent_list:
+      - package: 'golang-*'
```

Comment:
There is no `golang` package...

    root@gitlab-test1:~# COLUMNS=80 dpkg -l go*
    Desired=Unknown/Install/Remove/Purge/Hold
    | Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
    |/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
    ||/ Name           Version      Architecture Description
    +++-==============-============-============-=================================
    un  go-compiler    <none>                    (no description available)
    ii  golang-go      2:1.3.3-1~bp amd64        Go programming language compiler
    ii  golang-go-linu 2:1.3.3-1~bp amd64        Go standard library compiled for 
    un  golang-go.tool <none>                    (no description available)
    ii  golang-src     2:1.3.3-1~bp amd64        Go programming language compiler 
    un  golang-tools   <none>                    (no description available)
    un  golang-weekly- <none>                    (no description available)
    un  golang-weekly- <none>                    (no description available)
    un  golang-weekly- <none>                    (no description available)